worker/src/RTC/PortManager.cpp

```diff
@@ -217,6 +217,14 @@ namespace RTC
 			{
 				case Transport::UDP:
 				{
+#if defined(__linux__)
+					if (enableMulticast && Utils::IP::IsMulticast(&bindAddr, family))
```

Comment:
Let's use modern C++:

```suggestion
					if (enableMulticast && Utils::IP::IsMulticast(std::addressof(bindAddr), family))
```